src/test/java/se/nackademin/CartTest.java

```diff
@@ -0,0 +1,50 @@
+package se.nackademin;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import org.junit.Test;
+
+public class CartTest {
+    
+    @Test
+    public void shouldAddItemToCart(){
+        String[] currenctCart = new String[]{"item1","item2","item3"};
+        String newItem = "item4";
+
+        String[] exp = new String[]{"item1","item2","item3","item4"};
+        String[] output = Cart.addItemToCart(currenctCart, newItem);
+        assertArrayEquals(exp, output);
+    }
+
+    @Test
+    public void shouldNotAllowAddNullToCart(){
+        String[] currenctCart = new String[]{"item1","item2","item3"};
+        String newItem = null;
+
+        String[] exp = new String[]{"item1","item2","item3","NaN"};
+        String[] output = Cart.addItemToCart(currenctCart, newItem);
+        assertNotNull(exp[exp.length-1], output[output.length-1]);
```

Comment:
Not sure this makes complete sense. First, if a null value is provided, it would make more sense to ignore it and not add it, than to add a "NaN" string. Secondly, you use the assertNotNull where the first argument is the message to send on error and the second arg is the object to check if not null. But you send in expected and output instead.